### ISSUE TITLE
press enter after selecting option in settings menu to apply

### ISSUE THREAD

**Issue by jamesfranklinnetsec** [microsoft/terminal#10209]
the problem:

currently every time i go change the colour scheme of my profile because i walk into an area with different lighting and need different contrast, i have to laboriously click the colour scheme dropdown, select my scheme i want to try, then drag my mouse over to the apply button, then switch tabs back to that profile to see if it suits my lighting, which it might not, in which case i have to go do that process again.

naturally it would be nice to use keyboard shortcuts for most of this: 
1: ctrl+shift+tab to switch between settings and my terminal window
2: mouse is already over the dropdown
3: use arrow keys to select colour profile
4: >>>>press enter to apply<<<<
5: ctrl+shift+tab to switch back and check the colour scheme works

having it that way wouldnt require me to move my mouse at all, very convenient.

simple solution:
pressing enter applies changes in settings menu

# Description of the new feature/enhancement

# Proposed technical implementation details (optional)

**Comment by DHowett** [comment]
You might be a LOT happier with the "instant, temporary, per-terminal" version of this...

1. Press <kbd>Ctrl+Shift+P</kbd>
2. Search for "set color scheme" (or "col" or "setsch" or whatever; adjust for your language)
3. Choose a scheme with the arrows and hit enter.

The Settings UI will eventually simply autosave, and not require you to actually hit Save.

**Comment by jamesfranklinnetsec** [comment]
> You might be a LOT happier with the "instant, temporary, per-terminal" version of this...
> 
> 1. Press Ctrl+Shift+P
> 2. Search for "set color scheme" (or "col" or "setsch" or whatever; adjust for your language)
> 3. Choose a scheme with the arrows and hit enter.
> 
> The Settings UI will eventually simply autosave, and not require you to actually hit Save.

yeah that is very nice to know, slightly faster than tabbing between the settings and clicking, but would be nice to have a shortcut just to quickly open colour scheme-- maybe ctrl+alt+shift+c to open that menu with the dropdown. is there a way to set up shortcuts like that at the moment?

**Comment by zadjii-msft** [comment]
You know, there's been discussion in the past about allowing people to bind keys to a nested action, that would open the palette to the entries listed below it, but I'm not seeing that tracked anywhere officially. Congratulations, _this is now that thread_.

**Comment by FWest98** [comment]
I have been looking for this as well: I would like a way to immediately get to the `new tab...` page where I can search in my list. The existing command `openNewTabDropdown` does not allow search, unfortunately. A functionality similar to `tabSearch` would be much more user-friendly and would be achieved by implementing this feature.

**Comment by BWWIT** [comment]
So, is it still not possible to quickly open a profile by a dropdown-search?
As the functionality you get with the `New Tab...`
![image](https://github.com/microsoft/terminal/assets/151644805/709452ce-dbd5-4393-b13c-73fe7c897fff)

Tried to make a shortcut to `New Tab...` but didn't succeed :-(

A lot of people has a lot of profiles (hosts) and need at quick way to open these, please

**Comment by FWest98** [comment]
You can get around this now by using PowerToys Run. It has a plugin for Windows Terminal profiles, so I can now quickly start any WT profile using <kbd>Alt</kbd>+<kbd>Space</kbd>, then "wt [profilename]", <kbd>Enter</kbd>.
